src/Context/FeatureContext.php

```diff
@@ -822,7 +822,11 @@ public static function remove_dir( $dir ) {
 	 * Copy a directory (recursive). Destination directory must exist.
 	 */
 	public static function copy_dir( $src_dir, $dest_dir ) {
-		Process::create( Utils\esc_cmd( 'cp -r %s/* %s', $src_dir, $dest_dir ) )->run_check();
+		if ( 'Darwin' === PHP_OS ) {
+			Process::create( Utils\esc_cmd( 'cp -R %s/* %s', $src_dir, $dest_dir ) )->run_check();
+		} else {
+			Process::create( Utils\esc_cmd( 'cp -r %s/* %s', $src_dir, $dest_dir ) )->run_check();
+		}
```

Comment:
Not feeling too strongly about this, but I'd separate the shell command (the thing that changes based on platform) from its execution (the thing that always remains the same).
```suggestion
		$shell_command = ( 'Darwin' === PHP_OS )
			? Utils\esc_cmd( 'cp -R %s/* %s', $src_dir, $dest_dir )
			: Utils\esc_cmd( 'cp -r %s/* %s', $src_dir, $dest_dir );
		Process::create( $shell_command )->run_check();
```